tests/conform-yup.spec.ts

```diff
@@ -79,11 +104,48 @@ describe('conform-yup', () => {
 				min: 1,
 				max: 10,
 			},
-			timestamp: {},
-			options: {},
-			nested: {},
-			list: {},
-		});
+			timestamp: {
+				required: true,
+				min: new Date(1).toISOString(),
+				max: maxDate.toISOString(),
+			},
+			options: {
+				required: true,
+				multiple: true,
+			},
+			'options[]': {
+				required: true,
+				pattern: 'a|b|c',
+			},
+			nested: {
+				required: true,
+			},
+			'nested.key': {
+				required: true,
+			},
+			list: {
+				required: true,
+				multiple: true,
+			},
+			'list[]': {
+				required: true,
+			},
+			'list[].key': {
+				required: true,
+			},
+			tuple: {
+				required: true,
+			},
+			'tuple.0': {
+				required: true,
+				minLength: 3,
+			},
+			'tuple.1': {
+				max: 100,
+			},
```

Comment:
I am not sure how tuple works. There is no dedicated logic for it. Does it depends on your implementation of the `tuple` method above? Also, tuple has to be in the format `tuple[0]` / `tuple[1]`. 

It might be better leaving tuple unsupported 🤔

Reply:
agree. 👍  Thanks for the feedback. Tuple will be natively supported once `yup` version is updated to `v1`, so you can use that https://github.com/jquense/yup/issues/528#issuecomment-916885944